setup: dist/doc/html/tasty-golden/haddock-prolog10293.txt: invalid argument
diff --git a/tasty-golden.cabal b/tasty-golden.cabal
index aa61a0a..70292f8 100644
--- a/tasty-golden.cabal
+++ b/tasty-golden.cabal
@@ -5 +5 @@ description:
-  This package provides support for «golden testing».
+  This package provides support for "golden testing".
@@ -9 +9 @@ description:
-  «golden» file, which contains the correct result for the test.
+  "golden" file, which contains the correct result for the test.
